daemon/logger/logger.go

```diff
@@ -15,4 +21,6 @@ type Logger interface {
 	Log(*Message) error
 	Name() string
 	Close() error
+	GetReader() (io.Reader, error)
+	LogPath() string
```

Comment:
Short term okay for me. LogPath can be used by some tools like kubernetes, but we can deprecate it, because it's not a big deal to find it without this.